script.js

```diff
@@ -1,12 +1,187 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let allEpisodes;
+let allTVShows;
+let selectedShow;
+let allGenres;
+
+const searchShowsEl = document.querySelector('#searchShows');
+const searchEpisodesEl = document.querySelector('#searchEpisodes');
+const searchResult = document.querySelector('#search-result');
+const selectEpisodesEl = document.querySelector('#select');
+const selectShowEl = document.querySelector('#TVShows');
+const episodesEl = document.querySelector('#episodes');
+const showsEl = document.querySelector('#tvSeries');
+
+const getEpisodeTitle = obj => 'S' + `${obj.season}`.padStart(2, 0) + 'E' + `${obj.number}`.padStart(2, 0) + ` - ${obj.name}`;
+
+const getEpisodeCard = obj => `<div class="col-md-3 m-1 card">
+  <div class="card-body card-css">
+    <h5 class="card-title text-center">${getEpisodeTitle(obj)}</h5>
+    <hr>
+      ${getAndShowImage(obj)}
+    <p class="card-text">${obj.summary}</p>
+  </div>
+</div>`;
+
+const getShowCard = obj => `<div class="card col-12 col-md-10 m-3 show-card g-0">
+    <div class="row g-0">
```

Comment:
Remember about your [semantic HTML](http://html5doctor.com/downloads/h5d-sectioning-flowchart.pdf) from a few modules ago. Does it always make sense to use a div?